CONTRIBUTING.md

```diff
@@ -1,7 +1,122 @@
-# Contributing to a Rust Foundation Project
+# Contributing to the coding guidelines
 
-Thank you for your interest in contributing to this Rust Foundation project. 
-We are happy and excited to review and accept your pull requests.
+## Table of Contents
+
+- [Contributing to the coding guidelines](#contributing-to-the-coding-guidelines)
+  - [Table of Contents](#table-of-contents)
+  - [Contribution Workflow](#contribution-workflow)
```

Comment:
I think GH handles TOC automatically now in the README, so I think we can delete this whole section and the maintenance burden?

From main:

<img width="895" height="611" alt="Screenshot 2025-11-13 at 13 10 18" src="https://github.com/user-attachments/assets/c132773d-830a-4091-ba4f-dcb372c7d674" />